src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
 discuss: 2046
```

Comment:
@ydimova can you also add a bio for yourself to be included at the bottom of the chapter? (see TODO above)